pkg/generators/models/testdata/cases/oneof_mapped_discriminator/generated/model_container.go

```diff
@@ -21,6 +21,8 @@ func (m Container) Validate() error {
 	return validation.Errors{
 		"error": validation.Validate(
 			m.Error,
+			ErrorValidation{},
+			validation.Skip,
```

Comment:
what does the skip achieve? i thought it only skipped subsequent rules? does that mean skip the `m.Error.Validate()`?

Reply:
I think it skips all remaining rules and also the extra steps (so the usual nil check and the types own validation): https://github.com/go-ozzo/ozzo-validation/blob/v3.6.0/validation.go#L51 .